arcor2_AREditor/Assets/TABLET/Scripts/GUI/ActionObjectPickerMenu.cs

```diff
@@ -77,7 +77,7 @@ public enum CollisionObjectType {
             }
             // create one button for each object type
             foreach (ActionObjectMetadata actionObject in ActionsManager.Instance.ActionObjectsMetadata.Values.OrderBy(x => x.Type)) {
-                if (actionObject.Abstract || actionObject.CollisionObject)
+                if (actionObject.Abstract || (actionObject.CollisionObject && actionObject.ObjectModel.Type != ObjectModel.TypeEnum.Mesh))
```

Comment:
Maybe, this would deserve a comment explaining why it is like this. It makes sense to me that there are no buttons for abstract objects, but why not for collision objects?

Reply:
Yeah, good idea, it also took me a while to realize why is that so... :smile: I will add the comment.

Reply:
comment added